src/app/views/builder/builder.component.ts

```diff
@@ -0,0 +1,201 @@
+import {
+  Component,
+  ElementRef,
+  OnInit,
+  OnDestroy,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import {
+  CdkPortalOutlet,
+  ComponentPortal,
+  DomPortal,
+  TemplatePortal,
+} from '@angular/cdk/portal';
+import {
+  CdkDragDrop,
+  moveItemInArray,
+  copyArrayItem,
+} from '@angular/cdk/drag-drop';
+import {
+  stylesSheet_Textarea,
+  stylesSheet_Checkbox,
+  stylesSheet_Btn,
+  stylesSheet_Select,
+  stylesSheet_Input,
+} from '../../shared/style.sheets';
+
+import { StylePanelComponent } from '../../style-panel/style-panel.component';
+import { Store } from '@ngrx/store';
+import { getStyle, getGeneralStyle } from '../../core/store/index';
+import { AddStyleAction } from '../../core/store/styles-fields/styleFields.actions';
+import { FormControl, FormGroup } from '@angular/forms';
+import { valueDefault } from '../../shared/value.sheets';
+
+@Component({
+  selector: 'app-cdk-portal',
+  templateUrl: './builder.component.html',
+  styleUrls: ['./builder.component.scss'],
+})
+export class CdkPortalComponent implements OnInit, OnDestroy {
+  stylesSheet_Textarea = stylesSheet_Textarea;
+  stylesSheet_Checkbox = stylesSheet_Checkbox;
+  stylesSheet_Btn = stylesSheet_Btn;
+  stylesSheet_Select = stylesSheet_Select;
+  stylesSheet_Input = stylesSheet_Input;
```

Comment:
use camelCase as a namespace convention
```suggestion
  stylesSheet_Textarea = stylesSheet_Textarea;
  stylesSheet_Checkbox = stylesSheet_Checkbox;
  stylesSheet_Btn = stylesSheet_Btn;
  stylesSheet_Select = stylesSheet_Select;
  stylesSheet_Input = stylesSheet_Input;
```